python/README.md

```diff
@@ -1,8 +1,51 @@
 # Setup
 
-# Env
-- DON'T use conda as it prefers it's on ld path and libstd etc
-- Remaining instructions are for Ubuntu only
+Lance is a columnar file format designed for managing large-scale computer vision datasets in cloud-native, production
+environments. Lance delivers blazing fast performance for image and video data use cases from analytics to point 
+queries to training scans. 
+
+## Why use Lance
+
```

Comment:
Should we put this piece to the root README.md?

Reply:
good idea